src/guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -36,23 +36,22 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.6, 5.7
+*  MySQL 5.7 for on-premise installations
```

Comment:
shouldn't this be "on-premises" ? 😇

Reply:
I think [this post](https://www.adamfowlerit.com/2017/04/premise-vs-premises-cares/#:~:text=Yes%2C%20the%20word%20%E2%80%9Cpremises%E2%80%9D,considered%20in%20an%20official%20context.&text=so%2C%20it%20makes%20sense%20to,the%20correct%20term%20to%20use.) sums up my feelings about that.

Adobe uses "on-premise" as well, so I'm inclined to stick with that: https://helpx.adobe.com/legal/product-descriptions/adobe-experience-manager-on-premise.html.

Reply:
This did come up, and I researched it. "On-premises is in fact correct.  See https://collectivecontent.co.uk/2018/04/19/should-i-say-on-premise-or-on-premises-it/  

 Google search results:  
on-premises: 205,000,000
on-premise 86,900,000 

Thanks for remembering @hguthrie.  Also, it's not a show-stopper, but it's nice to do things correctly when we can.